.github/workflows/deploy-pr.yml

```diff
@@ -0,0 +1,29 @@
+name: deploy website to netlify
+
+on: [pull_request]
+
+jobs:
+  build-static-page:
+    name: build
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v1
+      - name: build image
+        run: |
+          docker build -t documentation:latest .
+          docker run -v ${PWD}:/output documentation:latest cp -r /usr/share/nginx/html /output/_site
```

Comment:
Does the docker version running have Buildkit support? If so, we could use `DOCKER_BUILDKIT=1 docker build --output ...`

Reply:
I think so, it actually is the moby 3.0.6(?) compiled by MSFT

Reply:
We'll probably need to modify the Dockerfile to have a specific target for that (buildstage "from scratch" with only the content)